arches_for_science/views/download_project_files.py

```diff
@@ -40,17 +71,24 @@ def post(self, request):
             }
             return JSONErrorResponse(content=response)
 
-    def create_download_zipfile(self, user, files, project_name):
+    def create_download_zipfile(self, user, files, temp_files, project_name):
         file_ids = [file["fileid"] for file in files]
         file_objects = list(models.File.objects.filter(pk__in=file_ids))
-        for file in files:
-            for file_object in file_objects:
-                if str(file_object.fileid) == file["fileid"]:
+
+        temp_file_ids = [temp_file["fileid"] for temp_file in temp_files]
+        temp_file_objects = list(models.TempFile.objects.filter(pk__in=temp_file_ids))
+
+        all_files = files + temp_files
+        all_file_objects = file_objects + temp_file_objects
+
+        for file in all_files:
+            for file_object in all_file_objects:
+                if str(file_object.fileid) == str(file["fileid"]):
                     file["file"] = file_object.path
         download_files = []
         skipped_files = []
         size_limit = 104857600  # 100MByte
-        for file in files:
+        for file in all_files:
             if file["file"].size >= size_limit:
```

Comment:
I discovered that this will break if the file cannot be found. Could you wrap this if/else in a try block and log an error message? Something like: logger.warn(_("Unable to locate {}".format(file["file"]))?